packages/medusa/src/api/routes/admin/uploads/get-download-url.ts

```diff
@@ -0,0 +1,34 @@
+import { IsString } from "class-validator"
+import { AbstractFileService } from "../../../../interfaces"
+import { validator } from "../../../../utils/validator"
+
+/**
+ * [get] /uploads
+ * operationId: "GetUploadsFileDownloadUrl"
+ * summary: "Gets a presigned download url for a file"
+ * description: "Gets a presigned download url for a file"
+ * x-authenticated: true
+ * parameters:
+ *   - (path) fileKey=* {string} key of the file to obtain the download link for.
```

Comment:
very nit: Match actual params
```suggestion
 *   - (path) file_key=* {string} key of the file to obtain the download link for.
```

Reply:
See other comment.